interactions/api/models/guild.py

```diff
@@ -3036,6 +3037,38 @@ def discovery_splash_url(self) -> Optional[str]:
             else None
         )
 
+    @classmethod
+    async def get_all_guilds(cls, client: "Client") -> List["Guild"]:
+        """
+        .. versionadded:: 4.4.0
+
+        Gets all guilds that the bot is present in.
+
+        :param Client client: The bot instance
+        :return: List of guilds
+        :rtype: List[Guild]
+        """
+
+        _after = None
+        _all: list = []
+
+        res: list = await client._http.get_self_guilds(limit=200)
+
+        while len(res) >= 1000:
```

Comment:
Just a question: why 1000?

Reply:
Copy paste from get all bans I guess

Reply:
you're right, wrong copy paste oops, fixed!